Worker.Extension.Sql/README.md

```diff
@@ -0,0 +1,180 @@
+# SQL Extension .NET Worker
+
+Welcome to the Sql Extension .NET Worker Repository. The .NET Worker provides .NET 6 support for SQL Bindings in Azure Functions, introducing an **Isolated Model**, running as an out-of-process language worker that is separate from the Azure Functions runtime. This allows you to have full control over your application's dependencies as well as other new features like a middleware pipeline. A .NET Isolated function app works differently than a .NET Core 3.1 function app. For .NET Isolated, you build an executable that imports the .NET Isolated language worker as a NuGet package. Your app includes a [`Program.cs`](vscode-file://vscode-app/c:/Users/makoripa/AppData/Local/Programs/Azure%20Data%20Studio/resources/app/out/vs/code/electron-browser/workbench/samples/samples-outofproc/Program.cs) that starts the worker.
```

Comment:
Remove the local path hardcoded in this - and it shouldn't be a vscode-file URL either